cmd/govim/config/config.go

```diff
@@ -80,6 +80,13 @@ type Config struct {
 	// comma-separated list
 	GoImportsLocalPrefix *string `json:",omitempty"`
 
+	// CompletionBudget is the soft latency string-format time.Duration goal for
+	// gopls completion requests. Most requests finish in a couple milliseconds,
+	// but in some cases deep completions can take much longer. As we use up our
+	// budget we dynamically reduce the search scope to ensure we return timely
+	// results. Zero seconds means unlimited
```

Comment:
Good idea. Done

Reply:
Actually, I'm going to row back on that slightly. We don't really want people to be configuring this value. Ability to change the value was initially added so that we could ensure integration tests didn't fail because of timeouts where we end up running ~GOMAXPROCS instances of Vim, `govim` and `gopls` concurrently as fast as possible. I'll add an example value but won't allude to what the default value is.